tests/unit/unit_tests_common.c

```diff
@@ -17,10 +17,12 @@
  * when the caller sets the VERBOSE env var to opt-in.
  */
 void
-set_log_streams_for_tests()
+set_log_streams_for_tests(msg_level exp_msg_level)
 {
-   if (Ctest_verbose) {
+   if (Ctest_verbosity == exp_msg_level) {
```

Comment:
Qs. Do you want to change this to `>=` here?

If we did that, then, when someone runs: `VERBOSE=7 ./bin/unit/misc_test`, they will start seeing error messages.

It will not be an issue for normal usages and for CI, but that's what will happen.

Let's decide which way to go ... so I can finalize the check here.

Reply:
Yes, `>=` is the normal semantic.

Reply:
Changed accordingly. See latest upload.